### PR TITLE
Don't check for locks on dequeue

### PR DESCRIPTION
**Pull request by srijeet0406**
This PR is not related to any issue. It removes the checking for CDN locks when trying to dequeue updates.

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?

Run TO and TP locally.
As user X, lock CDN A.
Queue updates on CDN A as user X.
Then as user Y, try clearing these updates on CDN A, while it is still locked by X.
This should succeed.

Make sure all unit/ API tests pass.

## If this is a bugfix, which Traffic Control versions contained the bug?

- master

## PR submission checklist
- [x] This PR has tests 
- [ ] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

**Comment by ocket8888** [comment]
Should this only ignore *soft* locks for dequeue? A hard lock is supposed to prevent editing servers, right? Queueing updates will change two fields (`lastUpdated` and `updPending` - once the changes in #6569 are merged it'll also change the update timestamp field)

**Comment by srijeet0406** [comment]
> Should this only ignore _soft_ locks for dequeue? A hard lock is supposed to prevent editing servers, right? Queueing updates will change two fields (`lastUpdated` and `updPending` - once the changes in #6569 are merged it'll also change the update timestamp field) on affected servers.

Resolved offline.

File: traffic_portal/app/src/common/modules/dialog/select/lock/dialog.select.lock.tpl.html [hunk 1]
```diff
@@ -51,7 +51,7 @@ <h4 class="modal-title">Create CDN Lock</h4>
                         </label>
                         <input id="soft" name="lockType" type="radio" ng-model="lock.soft" ng-value="true" required>
                         <label class="has-tooltip" for="hard">Hard<div class="helptooltip">
-                            <div class="helptext">A <dfn>hard</dfn> lock will prevent others from queueing server updates or snapshotting the locked CDN. It will also block others from making changes that would change the state of the locked CDN.
+                            <div class="helptext">A <dfn>hard</dfn> lock will prevent others from queueing server updates or snapshotting the locked CDN. It will also block others from making changes(except clearing updates) that would change the state of the locked CDN.
```
Comment (ocket8888) [inline traffic_portal/app/src/common/modules/dialog/select/lock/dialog.select.lock.tpl.html:54]: There should be a space between `changes` and `(except`

**Review by ocket8888 — CHANGES_REQUESTED** [review]
Assuming this works - I'm about to test - it's pretty much good to go.

**Review by ocket8888 — APPROVED** [review]

**Pull request merged**